docs/README.md

```diff
@@ -0,0 +1,30 @@
+This guide will demonstrate how to regenerate the plots used in the documentation here.
+The requirements are
+1) Have docker installed
+2) Download the [PlotData] zip file containing the scenarios used
+
+Next, extract the data to your home directory, or change the expected location in the
+docker-compose.yml file.
+
+
+Start the container, which will provide a notebook environment for running the code
+snippets.
+```
+docker compose up
+```
+
+After copying the notebook url from the output, start the `run_snippets.ipynb` notebook
+and run through it. For the most part, no interaction is required. The exception is any
+bokeh plots which must be manually saved as png (currently only the powerflow snapshot). 
```

Comment:
I think we can use the following:
```
from bokeh.io import export_png

export_png(plot, filename="plot.png")
```
to automatically save the plot as png

Reply:
Couple of issues.. we can get the plot object to pass in using [curstate()](https://docs.bokeh.org/en/latest/docs/reference/io.html#bokeh.io.state.curstate) if `output_notebook()` was called too, but the export also required `pip install selenium`, and after that I got an error that firefox isn't supported. At that point I gave up.